### PR TITLE
VN:MN: Moved diff from billing to hospitalcore

### PR DESCRIPTION
**Pull request by ndisha**
@geoff-wasilwa see changes done by @ngarivictor as per your earlier comments.
Solution to issue #1

### CODE REVIEW

File: omod/src/main/resources/sqldiff.xml [hunk 1]
```diff
@@ -262,6 +262,625 @@
...
+			ALTER  TABLE indoor_patient_service_bill_item add COLUMN `order_type` varchar(50) NULL;
+			ALTER TABLE billing_patient_service_bill_item ADD COLUMN voided_by varchar(255) DEFAULT Null;
+			ALTER TABLE indoor_patient_service_bill_item ADD COLUMN voided bit(1) DEFAULT Null;
+            ALTER TABLE indoor_patient_service_bill_item ADD COLUMN voided_date datetime DEFAULT Null;
+            ALTER TABLE indoor_patient_service_bill_item ADD COLUMN voided_by varchar(255) DEFAULT Null;
+		</sql>
+	</diff>
+		
 	<diff>
 		<version>2.2</version>
```
Comment (geoff-wasilwa) [inline omod/src/main/resources/sqldiff.xml]: @ndisha and @ngarivictor, this should be updated to `2.3.5`

**Comment by geoff-wasilwa** [comment]
Was this tested locally to see if the changes solved the dependency?

**Comment by ndisha** [comment]
@ngarivictor any progress with the error you were getting? Have you communicated the issue to @geoff-wasilwa?

**Comment by geoff-wasilwa** [comment]
@ndisha yes, we are working on it.

**Comment by geoff-wasilwa** [comment]
Closes #1

**Pull request merged**